client/src/components/Followers.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState ,Fragment } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { List, ListItem, ListItemText, Divider, ListItemAvatar, Avatar, Button} from "@material-ui/core";
+
+
+const useStyles = makeStyles((theme) => ({
+    list:{
+        padding:'0'
+    },
+    listItem:{
+        padding: '20px 25px'
+    },
+    unFollowButton:{
+        borderRadius: "10rem",
+        backgroundColor: "#DF1B1B",
+        color: "white",
+        width: "6rem",
+        height: "2rem",
+        fontSize:'0.6rem'
+        
+    }
+    
+  }));
+const Followers =(props)=>{
+    const classes = useStyles();
+    return(
+        <List aria-label='followers list' className={classes.list}>
+            {props.followersList.map((person=>{
+              return(
+                <Fragment>
```

Comment:
You are getting a key error as React requires that any elements that are generated from an array, must be each supplied a unique key. This can be temporarily fixed by passing the map index to Fragment as a key, but the best way would be when we implement a unique id to each Follower to pass that instead as the key to the Fragment.